docs/changelog.rst

```diff
@@ -10,37 +10,34 @@ v0.10.0
 *rc0: 2023-02-07*
 
 New features
-
     - `#1485`_ Added support for `"h"` (hours) and `"m"` (minutes) in `librosa.display.TimeFormatter`.  *Vincent Lostanlen*
     - `#1495`_ Expanded options for bandwidth calculation in `librosa.segment.recurrence_matrix` and `librosa.segment.cross_similarity`. *Tom Xi*
     - `#1529`_ Accelerated utility functions for squared magnitude `librosa.util.abs2` and `librosa.util.phasor`. *Brian McFee*
     - `#1514`_ `librosa.stft` support for pre-allocated output arrays.  *Brian McFee*
     - `#1533`_ Accelerated `librosa.util.localmax` and `librosa.util.localmin`. *Brian McFee*
-    - `#1552`_ **EXPERIMENTAL** Support for just-intonation based variable-Q transforms `librosa.vqt` and functional just system (FJS) notation. *Brian McFee*
+    - `#1552`_ **EXPERIMENTAL** Support for just-intonation based variable-Q transforms `librosa.vqt` and functional just system (FJS) notation. *Brian McFee, Vincent Lostanlen*
```

Comment:
LOL, thanks

Reply:
Sorry about that one!  It's bound to happen when I visually scan the PR list for authors and don't think too hard. :\